src/toil/test/src/jobServiceTest.py

```diff
@@ -109,7 +109,7 @@ def runToil(self, rootJob, retryCount=1, badWorker=0.5, badWorkedFailInterval=0.
         options.logLevel = "INFO"
         
         options.retryCount = retryCount
-        #options.clean = "never"
+        options.clean = "never"
```

Comment:
Presumably these tests succeed on master which doesn't have this change. If that is true, then a) your changes introduced the error you quote above or b) your other changes fix a bug in Toil which was masked by this line being commented out (options.clean defaulting to 'onsucess').